templates/base/src/utils/Logger/logger.js

```diff
@@ -0,0 +1,125 @@
+
+import fetch from 'isomorphic-unfetch';
+
+/**
+ * USAGE: ->
+ * import Logger from '../utils/Logger';
+ *  call as :-
+ * Logger.error({ message: 'Here is the error',
+ *    event: {
+      name: 'event20', id: 657,
+      module: 'modA',
+      type: "load|interaction|redirect" ,
+      data: { a: "hhhh" },
+    },
+    error: {
+      code: 'ABC',
+      message: 'message',
+      operationName: 'testing'
+    },
+    service: {
+      name: 'serviceName',
+      path: 'servicePath',
+    } });
+ */
+
+/**
+ * @param {object} msg --> msg Object
+ * @param {string}  logLevel --> string (info, log, error)
+ * returns an object with log data to be sent in api
+ */
+const getLogObject = ({ msg, logLevel }) => {
+  if (process.browser) {
+    const { message, event: { name: eventName, id, module, type, data }, service: { name: serviceName, path }, error: { code, message: errMsg, operationName } } = msg || {};
```

Comment:
Don't like such huge deconstructs. Why can't we simply use better naming on the data passed and use ... to apply the object of available.

Reply:
done